src/model/Team.js

```diff
@@ -58,6 +58,9 @@ class Team {
             if (data.hasOwnProperty('name')) {
                 obj['name'] = ApiClient.convertToType(data['name'], 'String');
             }
+            if (data.hasOwnProperty('parent_team')) {
+                obj['parent_team'] = convertRelatedObjectToType(data['parent_team'], Team);
```

Comment:
@david-merge will this work for a model referencing itself? Or does it need to be some type of "self" equivalent

Reply:
Yes, this should be fine. Hard passed type and then just creating JSON Objects for them manually in APIClient.convertToType()